web/src/list_widget.ts

```diff
@@ -163,7 +189,7 @@ export function create($container, list, opts) {
         sorting_functions: new Map(),
         offset: 0,
         list,
-        filtered_list: list,
+        filtered_list: [],
```

Comment:
Yeah, we can be sure that `filtered_list` will have valid data before we access it via `get_current_list`, because we call `widget.clean_redraw` when we create a list widget which sets the correct data for `filtered_list`.
https://github.com/zulip/zulip/blob/ba54dfd83cc05f8c9fb0f602bb3d2576f35d6ef5/web/src/list_widget.ts#L476